### ISSUE TITLE
Support for other hospitals

### ISSUE THREAD

**Issue by lukesleeman** [Western-Health-Covid19-Collaboration/wh_covid19_app#108]
Currently the app supports just Western health, and in particular WH anaesthetists.   However there is obviously a lot of potential to help out other hospitals and medical staff.  For the time being we need to stay laser focused on shipping what WH needs right away.

However in the mid to long term we are going to need to think about how to support others.  There are a few tasks things which we would need to work out:

- From a technical point of view how to we achieve this?  Different whitebox versions of the app?  Some kind of picker on the main screen where you choose who you work for?  Do they fork out git repo?

- From a process point of view how do we connect and coordinate with other organisations.

**Comment by lukesleeman** [comment]
Tagging this as `Low Priority - Later`  we don't need to deal with it right now.

**Comment by workerbee22** [comment]
White labelling would be effort intensive, as I assume you mean cloning the app for multiple hospitals, thus multiple user groups, diverging content and multiple app versions/beta/alpha builds. 

I'd suggest as a first step instead making the app more generic so any Australian hospital staff would find the content useful. Then we keep a single app and the workload does not balloon out. This might mean reducing the Western Health branding quite a bit.

Implementation example steps are `strings.dart` file for all semi-permanent content eg. app title, headings etc. Makes it very easy to change them as we evolve the app. We just create a class called`Strings` then refer to `Strings.appTitle` and similar in code rather than hard coding.

Many other things we can do as well.

**Comment by greggmiller** [comment]
I agree, once we have a functional app, anyone it is useful for should have access. The technical process of adapting the software for other hospitals is beyond my skillset, so I can only talk a little on the medical aspects.

In terms of content, all the guidance and information is quite generic. The actually variations in the step by steps between healthcare institutions would be pretty small. Most of these variations would be down to equipment availability, logistical issues and opinions within a department. Unfortunately, when there is not one proven, evidence-based way of doing things in medicine there are always slight variations. This is both a blessing and a curse of anaesthesia; it gives us some creative license but also means we rarely have absolute agreement on a given procedure. The latter is particularly true for our situation of a fast-evolving pandemic where we are rapidly learning as we go along. 

However, that being said this app is not about providing a prescriptive message to staff Anywhere (WH or otherwise), it is about easy-to-follow cognitive aids to ensure we get the broad strokes right on both staff protection (PPE) and in our day to day patient care of patients with COVID-19. I will address these aims more in #113 😄 

WH branding is not something integral to the app being released. It is more born out of apple not publishing any COVID19 apps (understandably in the age of misinformation), unless they have some affiliation to a recognised health provider. So we can reduce the branding aspect if required 👍. 

I've shown the content to friends both at hospitals elsewhere in Australia and in the UK. They are really enthusiastic about it too. Already in its current format, it will be of use to others. The level of variation we are talking about between hospitals would be in the region of the addition of one checkbox or omission of one in a checklist. The subheadings in processes and checklist would remain the same between institutions. 
My colleagues in other hospitals, I will make sure to add them to the beta group once the first iteration is available. Their opinion and advice can help guide how to go forward on this one from a medical perspective.

TL;DR
The app you are all building will serve as an excellent functional tool for WH anaesthetists and crit. care doctors
We are keen for it be used to support any others 👍 
All medical content is suggestive rather than prescriptive. 
Other hospitals variations in resources and process would be minor 😄

**Comment by lukesleeman** [comment]
Hmmmm, based on the comments above I'm wondering if we can address this by:

1. Providing better onboarding screens to more clearly contextualise the app ( see issue #113 )
2. Tweaking copy to make it more generic.  Eg, for a step that calls for a particular device which not all hospitals have, we could add "(If available)" or "(optional)".  For a step or checklist items that not all hospitals do we could put something like "(If indicated by your departments guidelines)"

Perhaps the way to address this is once it's out in beta we start gathering more specific suggestions from other institutions about what changes they would need.
